test/e2e/cloud_manager/deploy_sharded_cluster_test.go

```diff
@@ -99,6 +99,7 @@ func TestDeployCluster(t *testing.T) {
 	t.Run("Watch", watchAutomation(cliPath))
 
 	t.Run("Shutdown", func(t *testing.T) {
+		t.Skip("Waiting for ManagedSharding API to get fixed") // CLOUDP-187091
```

Comment:
Created [CLOUDP-187164](https://jira.mongodb.org/browse/CLOUDP-187164) to re-enable the tests.